templates/result.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Wyniki pobierania</title>
+  </head>
+  <body>
+    <h1>Wyniki pobierania</h1>
+    <p>Plik audio: {{ filename }}</p>
```

Comment:
It doesn't allow us to check if file is really downloaded.

I can put any string in url like `https://borek.herokuapp.com/result/foobar.mp4` and it will show me that `foobar.mp4`

it should check it request file exists on our storage:
- return 404 if not
- link to download file if yes
(this check should be implemented in result method, not template)